protocol/Cargo.toml

```diff
@@ -2,8 +2,14 @@
 name = "protocol"
 version = "0.1.0"
 authors = ["Vincent Hanquez <vincent.hanquez@iohk.io>", "Nicolas Di Prima <nicolas.diprima@iohk.io>", "The Rust-Crypto Project Developers"]
-license = "MIT"
-description = "protocol"
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/input-output-hk/rust-cardano"
+homepage = "https://github.com/input-output-hk/rust-cardano/protocol#README.md"
+description = """
+Cardano network protocol. Provides implementation to fetch blocks from a remote and
+to send transactions to a remote.
```

Comment:
Cardano network protocol. Provides an implementation to fetch blocks from a remote **location/node** and
to send transactions to a remote **location/node**.